content/en/serverless/aws_lambda/troubleshooting.md

```diff
@@ -31,6 +31,8 @@ See this [comparison guide][6] to understand the benefits of using the Extension
 
 To continue using the Forwarder, see this [Forwarder troubleshooting guide][8] for more help.
 
+<div class="alert alert-info">The Datadog Lambda extension enables log collection by default. If you are migrating from the Forwarder to the Extension, ensure that you remove your log subscription. Otherwise, you may see duplicate logs.</div>
```

Comment:
```suggestion
<div class="alert alert-info">The Datadog Lambda extension enables log collection by default. If you are migrating from the Forwarder to the extension, ensure that you remove your log subscription. Otherwise, you may see duplicate logs.</div>
```